pom.xml

```diff
@@ -66,26 +65,26 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>script-security</artifactId>
-      <version>1.13</version>
+      <version>1.29</version>
```

Comment:
Why not [1.29.1](https://wiki.jenkins.io/display/JENKINS/Script+Security+Plugin#ScriptSecurityPlugin-Version1.29.1(July10,2017))?

Reply:
Since it's test scope I guess the security fix doesn't matter.

Reply:
Thank you for the minor version I didn't see before 👍